src/App.tsx

```diff
@@ -192,7 +192,9 @@ const App = () => {
   const onCloseReleaseNotes = useCallback(() => {
     setShowReleaseNotes(false);
     if (!electronStore.get('appConfig.lastSeenReleaseNotesVersion')) {
-      dispatch(handleWalkThroughStep(StepEnum.Next));
+      dispatch(handleWalkThroughStep({step: StepEnum.Next, collection: 'novice'}));
+    } else {
+      dispatch(handleWalkThroughStep({step: StepEnum.Next, collection: 'release'}));
```

Comment:
I wonder if this should be reset to `0` for each release? We'd be responsible for properly updating the content. Currently the only option would be to keep adding content to the collection, but then after your first update you get all the content of the previous releases which seems unintended.

Reply:
Yes, we need to update release collection content along with release notes.
If we release only a patch version with no features, probably we don't want to show anything at all. We need to discuss it with the team